tests/Feature/AnnouncementApiTest.php

```diff
@@ -346,12 +346,14 @@ public function testDeleteAnnouncementGivenCorrectAuthorAndIdExpectResponseDelet
 
         $url = route('badaso.announcement.delete', ['id' => $announcement->id]);
         $response = AuthHelper::asUser($this, $user)->json('DELETE', $url);
+        $checkAnnouncement = AuthHelper::asUser($this, $user)->json('DELETE', $url);
 
         $response->assertStatus(200);
         $response->assertJsonFragment([
             'id' => $announcement->id,
             'content' => 'old content',
         ]);
+        $checkAnnouncement->assertStatus(400);
```

Comment:
hmm it's kinda hard to understand the purpose of these codes, since the 400 response status does not directly indicate that the announcement is not in db anymore. Why don't we just assert that `Announcement::count` is 0, or that Announcement where id equals our `$announcement->id` is null?